src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
:heart: @blakeembrey (ts-node / tsconfig) @cartant / @smrq (tsify) @johnnyreilly (also on ts-loader) @sebastian-lenz (typedoc) 

Sorry for the mention, thought I'd let you know. Feel free to ignore 💟  :rose: